packages/build/src/steps/get.ts

```diff
@@ -9,9 +9,10 @@ import { preCleanup } from '../plugins_core/pre_cleanup/index.js'
 import { preDevCleanup } from '../plugins_core/pre_dev_cleanup/index.js'
 import { saveArtifacts } from '../plugins_core/save_artifacts/index.js'
 import { scanForSecrets } from '../plugins_core/secrets_scanning/index.js'
+import { CoreStep } from '../plugins_core/types.js'
 
 // Get all build steps
-export const getSteps = function (steps, eventHandlers) {
+export const getSteps = function (steps, eventHandlers?: any[]) {
```

Comment:
Looks like this could be typed, even though they're a bit messy

Reply:
I already typed so much somewhere had to be the cut. Next time.